packages/flutter_adaptive_scaffold/example/lib/adaptive_layout_demo.dart

```diff
@@ -200,6 +200,39 @@ class _MyHomePageState extends State<MyHomePage> {
               unSelectedLabelTextStyle: navRailTheme.unselectedLabelTextStyle,
             ),
           ),
+          Breakpoints.expanded: SlotLayout.from(
+            key: const Key('Primary Navigation Expanded'),
+            inAnimation: AdaptiveScaffold.leftOutIn,
+            builder: (_) => AdaptiveScaffold.standardNavigationRail(
+              selectedIndex: selectedNavigation,
+              onDestinationSelected: (int newIndex) {
+                setState(() {
+                  selectedNavigation = newIndex;
+                });
+              },
+              extended: true,
+              leading: const Row(
+                mainAxisAlignment: MainAxisAlignment.spaceAround,
+                children: <Widget>[
+                  Text(
+                    'REPLY',
+                    style: TextStyle(color: Color.fromARGB(255, 255, 201, 197)),
```

Comment:
I know it's a demo, but since this color is used in a lot of these, it would be good if it were made into a constant somewhere with a description.